lib/modules/versioning/ubuntu/index.ts

```diff
@@ -6,15 +6,81 @@ export const displayName = 'Ubuntu';
 export const urls = ['https://changelogs.ubuntu.com/meta-release'];
 export const supportsRanges = false;
 
+const distroInfo = `Ubuntu 4.10 "Warty Warthog"
+Ubuntu 5.04 "Hoary Hedgehog"
+Ubuntu 5.10 "Breezy Badger"
+Ubuntu 6.06 LTS "Dapper Drake"
+Ubuntu 6.10 "Edgy Eft"
+Ubuntu 7.04 "Feisty Fawn"
+Ubuntu 7.10 "Gutsy Gibbon"
+Ubuntu 8.04 LTS "Hardy Heron"
+Ubuntu 8.10 "Intrepid Ibex"
+Ubuntu 9.04 "Jaunty Jackalope"
+Ubuntu 9.10 "Karmic Koala"
+Ubuntu 10.04 LTS "Lucid Lynx"
+Ubuntu 10.10 "Maverick Meerkat"
+Ubuntu 11.04 "Natty Narwhal"
+Ubuntu 11.10 "Oneiric Ocelot"
+Ubuntu 12.04 LTS "Precise Pangolin"
+Ubuntu 12.10 "Quantal Quetzal"
+Ubuntu 13.04 "Raring Ringtail"
+Ubuntu 13.10 "Saucy Salamander"
+Ubuntu 14.04 LTS "Trusty Tahr"
+Ubuntu 14.10 "Utopic Unicorn"
+Ubuntu 15.04 "Vivid Vervet"
+Ubuntu 15.10 "Wily Werewolf"
+Ubuntu 16.04 LTS "Xenial Xerus"
+Ubuntu 16.10 "Yakkety Yak"
+Ubuntu 17.04 "Zesty Zapus"
+Ubuntu 17.10 "Artful Aardvark"
+Ubuntu 18.04 LTS "Bionic Beaver"
+Ubuntu 18.10 "Cosmic Cuttlefish"
+Ubuntu 19.04 "Disco Dingo"
+Ubuntu 19.10 "Eoan Ermine"
+Ubuntu 20.04 LTS "Focal Fossa"
+Ubuntu 20.10 "Groovy Gorilla"
+Ubuntu 21.04 "Hirsute Hippo"
+Ubuntu 21.10 "Impish Indri"
+Ubuntu 22.04 LTS "Jammy Jellyfish"`;
+
+const codenameToVersion = new Map<string, string>();
+const versionToCodename = new Map<string, string>();
+
+// const populateMaps = (match): string =>
+//   match.replace(/(\r?\n)/g, '$1// renovate-replace ');
+//
+// distroInfo.replace(/([\d]*\.\d*)* "(.*)/g, populateMaps);
+const regex = /([\d]*\.\d*)( LTS)? "(.*) /g;
```

Comment:
You can constrain this a little more:
```suggestion
const regex = /(\d{1,2}\.(?:04|10))(?: LTS)? "([\w\s])+"/g;
```

(This also changes the capture groups so ` LTS` isn't captured)